### PR TITLE
Initial

### PR DESCRIPTION
**Pull request by tswicegood**
_Not complete_

### CODE REVIEW

File: armstrong/apps/crm/base.py [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+class UserBackend(BaseBackend):
+    """
+    Backend for handling user events and sending them to the CRM.
+
+    Each method receives a ``user`` representing the ``User`` model
+    that the action was performed on.  It also receives a ``payload``
+    parameter that is the ``**kwargs`` received by the signal.
+    """
+
+    def created(self, user, payload):
```
Comment (dmclain) [inline armstrong/apps/crm/base.py]: I'm -1 on payload not just being a **kwargs, all we're doing is providing a thin wrapper on top of signals and messing with the semantics like this doesn't gain us anything.

Comment (tswicegood) [inline armstrong/apps/crm/base.py]: I'm +0 on it.  If there's strong opinion on it, we can change it.

Comment (tswicegood) [inline armstrong/apps/crm/base.py]: I've changed it as @niran is -1 as well.  I've kept the `payload` variable name, however.

Comment (tinio) [inline armstrong/apps/crm/base.py]: also prefer this just being a *_kwargs, so +1 on the change to *_payload.

**Pull request merged**